src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -309,6 +309,9 @@ public TestCase updateTestCase(
       validatedTestCase.setJson(
           QiCoreJsonUtil.updateResourceFullUrls(validatedTestCase, madieJsonResourcesBaseUri));
     }
+    validatedTestCase.setJson(
+        QiCoreJsonUtil.replacePatientRefs(
+            validatedTestCase.getJson(), validatedTestCase.getPatientId().toString()));
```

Comment:
I think this should be conditional just for QICore.

Reply:
moved into the bracket above!